javascript-conditionals/main.js

```diff
@@ -110,6 +110,8 @@ function recommendMovie(genre) {
       return 'La La Land';
     case 'sci-fi':
       return 'Blade Runner 2049';
+    default:
+      return 'Genre not found.';
```

Comment:
Compare your error message to the one in the instructions. This isn't the correct string that should be getting `return`ed